app/user/rest.py

```diff
@@ -53,7 +56,7 @@ def create_user():
 
 @user.route('/<uuid:user_id>', methods=['PUT'])
 def update_user(user_id):
```

Comment:
Is this endpoint marked for deletion? Once we have POST (See other comment) and an update that allows per field updating can we remove the PUT whole user endpoint?